spec/sony_ci_api/client_spec.rb

```diff
@@ -556,27 +563,27 @@ def self.config_hash
       invalid_config_file.close && invalid_config_file.unlink
 
       # Destroy our FakeCredentials object.
-      Object.send(:remove_const, :TestCredentials)
+      TestCredentials.clear
```

Comment:
The `.clear` method probably isn't needed. The call to `Object.remove_const` here is just obliterating any trace of the `TestCredentials` class once we leave the spec, but that level of cleanup is also probably unnecessary and just adds to confusion in this case. If `TestCredentials` could just be a hash, then that would be better, but a hash that is local to an RSpec context won't be available in ERB when loading a config file. So that's why i made it a class.

An alternative approach would be a globally accessible helper method, but it's sort of 6 of one, half dozen of the other at that point, imo.